dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsSupport.java

```diff
@@ -61,23 +62,42 @@ public class MetricsSupport {
     private static final String version = Version.getVersion();
     private static final String commitId = Version.getLastCommitId();
 
-    public static Map<String, String> applicationTags(ApplicationModel applicationModel) {
+    public static Map<String, String> applicationTags(ApplicationModel applicationModel, @Nullable Map<String, String> extraInfo) {
```

Comment:
Add a overwrite method `Map<String, String> applicationTags(ApplicationModel applicationModel)` to prevent use `null` for caller

Reply:
ok